src/forms/login-popup/index.jsx

```diff
@@ -221,10 +225,12 @@ const LoginForm = () => {
           </div>
         </>
       )}
-      <LoginFailureAlert
-        errorCode={errorCode.type}
-        context={errorCode.context}
-      />
+      <div ref={errorAlertRef}>
+        <LoginFailureAlert
+          errorCode={errorCode.type}
+          context={errorCode.context}
+        />
+      </div>
```

Comment:
[Question] will screen reader read the error if we apply ref to div? if yes, fine and if not, can we pass the ref inside loginFailureAlert?